questionnaire/questionnaire-service.test.js

```diff
@@ -192,26 +212,25 @@ function getQuestionnaireDefinition() {
     };
 }
 
-const questionnaireService2 = createQuestionnaireService({
-    logger: () => 'Logged from dataset test',
-    createQuestionnaireDAL: () => ({
-        getQuestionnaire: questionnaireId => {
-            if (questionnaireId === '01fa0d1e-000a-404c-8efe-7223c24a4fa7') {
-                return getQuestionnaireDefinition();
-            }
-
-            throw new VError(
-                {
-                    name: 'ResourceNotFound'
-                },
-                `Questionnaire "${questionnaireId}" not found`
-            );
-        }
-    })
-});
-
 describe('Questionnaire Service', () => {
     describe('Answering a section', () => {
+        const questionnaireService2 = createQuestionnaireService({
+            logger: () => 'Logged from dataset test',
+            createQuestionnaireDAL: () => ({
```

Comment:
Can this be Mocked and just naturally passed in via the dependency injection? Not explicitly passed in to the `createQuestionnaireService` function

Reply:
I didn't change this, only moved it. I'd rather avoid refactoring old tests where I can as these will be deprecated with v1.